# Define the goal.
sensor_msgs/Image image
---
# Define the result.
judge_target_object/BoundingBox2D[] bounding_boxes
---
# Define a feedback message.
